### PR TITLE
Quiz point calculation done

### PR DESCRIPTION
**Pull request by Nothingful**
Added companion object for quiz question point calculation.

### CODE REVIEW

**Comment by vhrvoje2** [comment]
Youre missing the implementation for multiple questions module

**Comment by Nothingful** [comment]
> 
> 
> You're missing the implementation for multiple questions module

Yeah, I noticed.
Fixed

File: CircuitMessing/MultipleQuestion/src/main/java/com/example/radioquestion/MultipleQuestion.kt [hunk 1]
```diff
@@ -16,6 +17,13 @@ public class MultipleQuestion(
     }
 
     override fun checkAnswers(): Boolean {
-        return true
+        Log.d("LOG Answer: ", MultipleChoiceQuestionFragment.userAnswer.toString())
+        Log.d("LOG Correct answer: ", CorrectAnswers[0])
+        var check: Boolean = false
+        for (answer in MultipleChoiceQuestionFragment.userAnswer) {
+            if (answer == CorrectAnswers[0]) check = true
```
Comment (vhrvoje2) [inline CircuitMessing/MultipleQuestion/src/main/java/com/example/radioquestion/MultipleQuestion.kt:24]: the idea is good but this needs to be inverted
set default boolean to true and if any questions is not in the correct answers list flip to false and you can even break out of the loop in that case

**Review by vhrvoje2 — CHANGES_REQUESTED** [review]
Multiple questions check is not done the right way

File: CircuitMessing/MultipleQuestion/src/main/java/com/example/radioquestion/MultipleQuestion.kt (hunk 1 above)
Comment (bzitkovic) [reply CircuitMessing/MultipleQuestion/src/main/java/com/example/radioquestion/MultipleQuestion.kt:24]: Yes. Should be fixed so I can save points and titles to the database before the sprint ends.

**Comment by bzitkovic** [comment]
Okay. Issues have been fixed now. Merging.

**Pull request merged**